### ISSUE TITLE
Script error

### ISSUE THREAD

**Issue by zarMarco** [HackeSta/polybar-kdeconnect#5]
Hi I'm going to try your script. When I insert module on my polybar config and I restarted polybar, my bars disappear

**Comment by haideralipunjabi** [comment]
This issue was originally mentioned on Reddit as [this comment](https://www.reddit.com/r/unixporn/comments/aot2mk/oc_polybarkdeconnect_v2_kdeconnect_module_for/eifk6n5)

I tried to solve the issue there, but as of now it is unsolved and open. I will add the highlights here if someone else has the same issue.

* The original bar disappearance solved itself
* After the bar appeared, it showed the following error `/bin/sh:show_devices:command not found`
* The problem was fixed by adding the missing space between `.` and `/path` in the line `exec = ". /path/to/files/polybar-kdeconnect.sh; show_devices" `. I have also added an extra note in README in dfe8a3d18db685cc83635bc90ee0a3a8050d5059

* The module shows no error, but no device appears in the bar.
* I believe it might be due to missing `qt5tools` dependency. I was originally waiting for #1 to solve, but have added the missing dependency to README in dfe8a3d18db685cc83635bc90ee0a3a8050d5059
* Also, KDE Connect Settings from the `kdeconnect` package can be used to check if any device is present on the network or not.
